packages/forms-web-app/src/csp.js

```diff
@@ -1,16 +1,25 @@
 const helmet = require('helmet');
 const crypto = require('node:crypto');
 
-const ga = 'https://www.google-analytics.com';
-const gar = 'https://region1.google-analytics.com';
-const gas = 'https://ssl.google-analytics.com';
-const gtm = 'https://www.googletagmanager.com';
-
-const scriptSrc = [ga, gtm, gas];
-
-const connectSrc = [gar, 'https://api.os.uk'];
-
-const imgSrc = ['https://cdn.jsdelivr.net/gh/OrdnanceSurvey/os-api-branding@0.3.1/img/'];
+const scriptSrc = ['https://*.googletagmanager.com'];
```

Comment:
`scriptSrc` include `https://www.google-analytics.com` (`ga`) and `https://ssl.google-analytics.com` (`gas`) previously. Maybe worth keeping those?

Reply:
as discussed not needed